src/github_runner_image_builder/config.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Module containing configurations."""
+
+import argparse
+import logging
+import platform
+from enum import Enum
+from pathlib import Path
+from typing import Literal
+
+from github_runner_image_builder.errors import UnsupportedArchitectureError
+
+logger = logging.getLogger(__name__)
+
+ACTION_INIT = "init"
```

Comment:
I'm not a big fan of having constants in a file to import them somewhere else. I think of this as an antipattern. In my opinion, you should decide whether to put them in the `builder`or the `cli` based on what provides more cohesion. For instance, `IMAGE_OUTPUT_PATH` should probably be defined in the `builder` and imported in the  `cli`

Reply:
This was a suggestion from @gregory-schiano and @jdkandersson so that we can keep track of the configuration values scoped to one file, so that maintenance becomes a little easier.
The reason for not defining in the `builder` and importing in `cli` is due to circular dependencies :)

Reply:
Understood